src/components/Inputs/Search/Search.docs.mdx

```diff
@@ -0,0 +1,51 @@
+import { Links } from 'src/storybook';
+import { Primary, Canvas } from '@storybook/blocks';
+import * as ComponentStories from './Search.stories';
+
+# Search
+
```

Comment:
Should this have the "Component from EDS" text?

Reply:
Yep, adding